.woodpecker/docs.yml

```diff
@@ -21,10 +21,10 @@ pipeline:
       - chmod 0600 $HOME/.ssh/id_rsa
       - cd docs/
       - yarn install --frozen-lockfile
+      - yarn build
       - yarn deploy
     when:
-      event: push
-      branch: master
+      event: [pull_request]
```

Comment:
Are you sure we should deploy on PRs? This was initially the job to update the docs on pushes to master. `yarn deploy` should build and deploy the docs to our github page.

Reply:
no it's for testing porpose - but didn't worked out